src/pages/ProjectEdit/index.tsx

```
@@ -0,0 +1,124 @@
+      navigate(`${routes.projectList}`);
+    } catch (error) {
+      showSnackbar("Error updating project:", "error");
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const handleChange = (
+    e: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ) => {
+    const { value, name } = e?.target;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)